smp/enumeration_management.py

```diff
@@ -0,0 +1,121 @@
+"""The Simple Management Protocol (SMP) Enumeration Management group."""
+
```

Comment:
Adding `from __future__ import annotations` here will hopefully fix the builds for old python.